_pages/imprint.html

```diff
@@ -3,39 +3,49 @@
 permalink: /imprint
 ---
 
-<br/><br/><br/>
+<br /><br /><br />
 
 <section class="section_padding_100 clearfix section_white">
-    <div class="container">
-        <div class="row">
-            <div class="col-md-12">
-                <h1>Impressum</h1>
-            </div>
-            <div class="col-md-6">
-                Diese App wurde von Studierenden in Kooperation und Leitung des Lehrstuhls f&uuml;r Betriebssysteme erstellt.
-                Für alle Anliegen erreichen Sie den Support unter:<br/>
-                <b>app@tum.de</b><br/>
-                Aktuelle Projektleitung erfolgt durch <a href="https://bruck.me">Kordian Bruck</a>. Die Entwicklung erfolgt
-                primär durch Freiwillige und wird öffentlich auf <a href="https://github.com/TUM-Dev/">GitHub</a> durchgeführt,
-                wo auch der Quelltext für alle Projekte zur Einsicht bereit liegt.<br/><br/>
+  <div class="container">
+    <div class="row">
+      <div class="col-md-12">
+        <h1>Impressum</h1>
+      </div>
+      <div class="col-md-6">
+        <h4>Verein</h4>
+        <b>Open Source @ TUM e.V.</b><br />
+        Vertreten durch: <br />
+        Frank Elsinga (1. Vorsitzender)<br />
+        Joscha Henningsen (2. Vorsitzender)<br />
+        <br />
+        <h5>Campus App</h5>
+        Die Campus App wurde von Studierenden in Kooperation und Leitung des
+        Lehrstuhls f&uuml;r Betriebssysteme erstellt. Für alle Anliegen
+        erreichen Sie den Support unter:<br />
+        <b>app@tum.de</b><br />
+        Aktuelle Projektleitung erfolgt durch
+        <a href="https://bruck.me">Kordian Bruck</a>. Die Entwicklung erfolgt
```

Comment:
@jakobkoerber should we replace this, given that you are the flutter lead? 😅 

```suggestion
        Jakob Körber. Die Entwicklung erfolgt
```